Bottom: 0b5048f0657ac0bdc687a2cbe47374df9940e20c
Top:    2125fd8718d6cfc3235e346c3c2a710e27d123f3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-11 15:31:12 +0000

Refresh of add-configure-checks-for-gtkgl

---

diff --git a/configure.ac b/configure.ac
index d8f230f..01f6780 100644
--- a/configure.ac
+++ b/configure.ac
@@ -802,7 +802,7 @@ AM_CONDITIONAL(DEBUG_BUILD, test x$enable_debug = xyes)
 # ------------- Complete set of CFLAGS and LIBS -------------------
 
 CFLAGS="$CFLAGS $X_CFLAGS $DBUS_CFLAGS $GTK_CFLAGS $GTKGLEXT_CFLAGS"
-LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GTK_LIBS $GTKGLEXT_LIBS $DMALLOC_LIBS $GD_LIBS $INTLLIBS -lglut"
+LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GTK_LIBS $GTKGLEXT_LIBS $DMALLOC_LIBS $GD_LIBS $INTLLIBS"
 
 
 # if we have gcc then add -Wall
